manager/controllers/default/header.php

```diff
@@ -303,10 +305,16 @@ public function hasPermission($perms)
      *
      * @return void
      */
-    public function processSubMenus(&$output, array $menus = array())
+    public function processSubMenus(&$output, array $menus = array(), int $maxItems)
```

Comment:
```suggestion
    public function processSubMenus(&$output, array $menus = array(), $maxItems = false)
```

MODX should be compatible with PHP 5.6 and higher, scalar type hints are available in PHP 7+ only. 
A default value should be used to maintain backwards compatibility.

Reply:
You're right